lms/templates/problem_ajax.html

```diff
@@ -1 +1 @@
-<div id="problem_${element_id}" class="problems-wrapper" data-problem-id="${id}" data-url="${ajax_url}" data-progress_status="${progress_status}" data-progress_detail="${progress_detail}"></div>
+<div id="problem_${element_id}" class="problems-wrapper" data-problem-id="${id}" data-url="${ajax_url}" data-progress_status="${progress_status}" data-progress_detail="${progress_detail}" data-content="${content | h}"></div>
```

Comment:
Partly laziness, partly risk aversion. The `render()` method already takes content as an optional arg, and I don't completely understand what it's doing in terms of binding and events when it does initialization during that render. I figured that this would be less likely to lead to odd bugs.